### PR TITLE
[fluentd-elasticsearch] allow plugins install

### PR DESCRIPTION
**Pull request by eddycharly**
Adresses issue #181

### CODE REVIEW

**Comment by monotek** [comment]
Containers should be imutable and therefore this is bad practice.

The image could change on container restart, if a newer plugin version is installed as before and the container start could fail entirely.

If you need to add stuff to the image you should build a new one.

**Comment by eddycharly** [comment]
I see your point.

I thought it could help reduce maintenance as it doesn't require to rebuild an image every time a new image of fluentd is available.

I don't know if it is possible to call gem install specifying a version though, it would ensure that the plugin version installed is always the same.

**Comment by axdotl** [comment]
Installing a specific version is possible via `gem install <GEMNAME> --version <VERSION>` (see <https://guides.rubygems.org/command-reference/#gem-install>)
There are other charts (e.g. stable/grafana) which also allow to install arbitrary plugins.

This means, I generally agree with both of you :) Which means that adding additional plugins is fine for me, as long as specifying a specific version is required.

What are you thoughts on this @monotek ?

**Comment by eddycharly** [comment]
I improved the `additionalPlugins` list to include a specific version.

**Comment by eddycharly** [comment]
@monotek @axdotl are you open to discuss this further ?

**Comment by davideagle** [comment]
I'd be interested in this feature and open for discussion on the topic

**Comment by axdotl** [comment]
Sorry for the delay. But we are currently on vacation and not that active.
I'll check it during the weekend.

**Comment by eddycharly** [comment]
@axdotl DCO and CI are all green.

File: charts/fluentd-elasticsearch/templates/daemonset.yaml [hunk 1]
```diff
@@ -54,6 +54,17 @@ spec:
       - name: {{ include "fluentd-elasticsearch.fullname" . }}
         image:  "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+        {{- if .Values.additionalPlugins }}
+        command:
+          - bash
```
Comment (axdotl) [inline charts/fluentd-elasticsearch/templates/daemonset.yaml]: pls. use full path to bash

Comment (eddycharly) [reply charts/fluentd-elasticsearch/templates/daemonset.yaml]: done

**Comment by axdotl** [comment]
Pls. signoff your last commit to pass the DCO check.
When its done, I'll merge.

**Comment by eddycharly** [comment]
@axdotl i'm updating the readme also, i will singoff the DCO right after.

**Comment by eddycharly** [comment]
@axdotl it should be good now.

**Comment by eddycharly** [comment]
@axdotl thanks for your support.

**Pull request merged**